TestValidateBlock test fails on develop

```
--- FAIL: TestValidateBlock (0.11s)
panic: Panicked on a Sanity Check: can't encode times below 1970 [recovered]
        panic: Panicked on a Sanity Check: can't encode times below 1970

goroutine 20 [running]:
testing.tRunner.func1(0xc4201940f0)
        /usr/local/go/src/testing/testing.go:711 +0x5d9
panic(0xbe4e40, 0xc42019de00)
        /usr/local/go/src/runtime/panic.go:491 +0x2a2
github.com/tendermint/tendermint/vendor/github.com/tendermint/tmlibs/common.PanicSanity(0xbe4e40, 0xe6cbe0)
        /go/src/github.com/tendermint/tendermint/vendor/github.com/tendermint/tmlibs/common/errors.go:26 +0x120
github.com/tendermint/tendermint/vendor/github.com/tendermint/go-wire.WriteTime(0x0, 0x0, 0x0, 0x7fce80c00998, 0xc420178b60, 0xc420129b40, 0xc42019dd50)
        /go/src/github.com/tendermint/tendermint/vendor/github.com/tendermint/go-wire/time.go:19 +0x11e
github.com/tendermint/tendermint/vendor/github.com/tendermint/go-wire.writeReflectBinary(0xcb9640, 0xc42018a490, 0x199, 0x11947c0, 0xcb9640, 0xbaa927, 0x9, 0x0, 0xcb9640, 0xc4200d5280, ...)
        /go/src/github.com/tendermint/tendermint/vendor/github.com/tendermint/go-wire/reflect.go:525 +0x22f0
github.com/tendermint/tendermint/vendor/github.com/tendermint/go-wire.writeReflectBinary(0xc2af40, 0xc42018a460, 0x16, 0x11947c0, 0xc8c5c0, 0x0, 0x0, 0x0, 0x0, 0x0, ...)
        /go/src/github.com/tendermint/tendermint/vendor/github.com/tendermint/go-wire/reflect.go:530 +0x21e7
github.com/tendermint/tendermint/vendor/github.com/tendermint/go-wire.WriteBinary(0xc2af40, 0xc42018a460, 0x7fce80c00998, 0xc420178b60, 0xc420129b40, 0xc42019dd50)
        /go/src/github.com/tendermint/tendermint/vendor/github.com/tendermint/go-wire/wire.go:80 +0x15f
github.com/tendermint/tendermint/vendor/github.com/tendermint/tmlibs/merkle.SimpleHashFromBinary(0xc2af40, 0xc42018a460, 0xa, 0xc4201941e0, 0xa)
        /go/src/github.com/tendermint/tendermint/vendor/github.com/tendermint/tmlibs/merkle/simple_tree.go:76 +0x184
github.com/tendermint/tendermint/vendor/github.com/tendermint/tmlibs/merkle.SimpleHashFromBinaries(0xc42018b900, 0xa, 0xa, 0xc42018b900, 0xa, 0xa)
        /go/src/github.com/tendermint/tendermint/vendor/github.com/tendermint/tmlibs/merkle/simple_tree.go:68 +0xe9
github.com/tendermint/tendermint/types.(*Commit).Hash(0xc42017c500, 0xc400000001, 0x1, 0x30)
        github.com/tendermint/tendermint/types/_test/_obj_test/block.go:406 +0x252
github.com/tendermint/tendermint/types.(*Block).FillHeader(0xc4200d5240)
        github.com/tendermint/tendermint/types/_test/_obj_test/block.go:115 +0x1ff
github.com/tendermint/tendermint/types.MakeBlock(0x3, 0xcc3a28, 0x5, 0xc42032b4d0, 0x2, 0x2, 0xa, 0xc42017c500, 0xc4200143e0, 0x20, ...)
        github.com/tendermint/tendermint/types/_test/_obj_test/block.go:48 +0x438
github.com/tendermint/tendermint/types.TestValidateBlock(0xc4201940f0)
        /go/src/github.com/tendermint/tendermint/types/block_test.go:22 +0x468
testing.tRunner(0xc4201940f0, 0xe0de38)
        /usr/local/go/src/testing/testing.go:746 +0x16d
created by testing.(*T).Run
        /usr/local/go/src/testing/testing.go:789 +0x569
exit status 2
FAIL    github.com/tendermint/tendermint/types  0.162s
make: *** [test_integrations] Error 1
```

--- a/types/block_test.go
+++ b/types/block_test.go
@@ -2,6 +2,7 @@ package types
 
 import (
 	"testing"
+	"time"
 
 	"github.com/stretchr/testify/require"
 	crypto "github.com/tendermint/go-crypto"
@@ -65,6 +66,7 @@ func makeCommit(blockID BlockID, height int64, round int,
 		Round:            round,
 		Type:             VoteTypePrecommit,
 		BlockID:          blockID,
+		Timestamp:        time.Now().UTC(),
 	}
 
 	// all sign

